document_page_approval/security/ir.model.access.csv

```diff
@@ -0,0 +1 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
This empty file makes no sense, I think the best thing to do is to delete it.

Reply:
I think this file is necessary for the tests

Reply:
It is not necessary and does nothing, remove it.